src/accessControl/acp/mockedClientApi.js

```diff
@@ -49,6 +49,10 @@ export function getRequiredRuleOnPolicyAll(policy) {
   throw new Error("Not implemented");
 }
 
+export async function getSolidDatasetWithAcp(url, options) {
```

Comment:
Small heads-up: this will be called `getSolidDatasetWithAcr` I think.

Reply:
Noted - will fix that in later PRs when we're connecting to the actual APIs.